packages/tools/devtools/devtools-example/src/test/ui-smoke-test/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"compilerOptions": {
+		"module": "esnext",
+		"outDir": "../dist/ui-smoke-test",
```

Comment:
The `outDir` should end up outside of `src/`, so the output files aren't seen as input files by the "regular" build the next time it runs.

That said, I even wonder if we should have a separate build for the tests in this package... the unit tests in `src/test` already get built as part of the single build we do right now for all the source code, and for an example app I think it's probably fine to have the jest test be part of that too. @Josmithr, thoughts? It means the single tsconfig gets chunky. needing all the types specified here, but I think it's ok for a sample app.

And if we do, the whole package should probably be ESM (since it's not library), and the single build be a `module: "esnext"` one.

Reply:
I see! I synced-up with Josh offline. We are planning to disable the `CounterWidget` test and only runs e2e tests because its test environment is conflict with jest pupeteer, and just keep `src/test` to have built jest tests.